crates/load-test/Cargo.toml

```diff
@@ -7,10 +7,10 @@ license = "MIT OR Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-goose = "0.16.3"
+goose = { git = "https://github.com/eqlabs/goose", branch = "update_tokio" }
```

Comment:
Ah, this might not be _that_ simple: https://github.com/tag1consulting/goose/issues/419

(I had a PR for goose previously before realizing that it indeed breaks with new tokio...)

Maybe we should just remove the load-test crate from the workspace and have it as a completely independent tool?

Reply:
I just ran those tests and you're absolutely right. Good catch :pray: .
Removing the tool from the workspace seems like the simplest solution for now.